cookbook/upgrading.rst

```diff
@@ -0,0 +1,127 @@
+How to Upgrade your Symfony Project
+===================================
+
+So a new Symfony release has come out and you want to upgrade, great! Fortunately,
+because Symfony protects backwards-compatibility very closely, this *should*
+be quite easy.
+
+Upgrading a Patch Version (e.g. 2.6.0 to 2.6.1)
+-----------------------------------------------
+
+If you're upgrading and only the patch version (the last number) is changing,
+then it's *really* easy:
+
+.. code-block:: bash
+
+    $ composer update symfony/symfony
+
+That's it! You should not encounter any backwards-compatability breaks or
+need to change anything else in your code.
+
+You may also want to upgrade the rest of your libraries. If you've done a
+good job with your version constraints in ``composer.json``, you can do this
+safely by running:
+
+.. code-block:: bash
+
+    $ composer update symfony/symfony
+
+But beware. If you have some bad version constraints in your ``composer.json``,
+(e.g. ``dev-master``), then this could upgrade some non-Symfony libraries
+to new versions that contain backwards-compability changes.
+
+Upgrading a Minor Version (e.g. 2.5.3 to 2.6.0)
+-----------------------------------------------
+
+If you're upgrading a minor version (where the middle number changes), then
+you should also *not* encounter significant backwards compability changes.
+For details, see our :doc:`/contributing/code/bc`.
+
+However, some backwards-compability breaks *are* possible, and you'll learn
```

Comment:
I don't think so - I think serial commas are only when you have lists (e.g. of 3+ items). But I can't explain *why* I want a comma here - it feels a little bit better, and I can't think of a rule that I'm using or breaking :)